operators/advanced_network/advanced_network/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Having nested `advanced_network/advanced_network` folders is confusing, it seems like the lower folder is not Holoscan specific, could we have `advanced_network/lib` instead?

Reply:
The reason is to be able to include `advanced_network/common.h` instead of `common.h`

Reply:
Thanks for the explanation, makes sense